hack/get_ci_vm.sh

```diff
@@ -47,11 +47,10 @@ elif [[ "$1" == "--setup" ]]; then
     cd $REPO_DIRPATH
     echo "+ Loading ./contrib/cirrus/lib.sh" > /dev/stderr
     source ./contrib/cirrus/lib.sh
-    echo "+ Mimicking .cirrus.yml clone_script and build_task" > /dev/stderr
+    echo "+ Mimicking .cirrus.yml build_task" > /dev/stderr
     make install.tools
-    make vendor
```

Comment:
Why is make vendor excluded?

Reply:
I didn't think it necessary in this context.  I want to get the developer into a VM as quickly as possible, this seemed like a needless step.  But what to I know, if you feel it's useful here I'll gladly put it back, no prob.

Reply:
(honestly, I do not know 100% it's not useful)

Reply:
You are right it is not needed here, I overlooked that this is get_ci_vm.sh.

Reply:
I don't think the make vendor is required.  There is a test to make sure that make vendor does not change anything.  If it does it could be a problem.  But that should be in the smoke test.